defaults/main.yml

```diff
@@ -1,7 +1,9 @@
 ---
 # Install from phar (faster, but less flexible).
 drush_version: "8.1.10"
-drush_phar_url: https://github.com/drush-ops/drush/releases/download/{{ drush_version }}/drush.phar
+# Install Drush Launcher (Drupal 8.4+).
+drush_launcher_version: "0.3.1"
```

Comment:
I suggest changing this to the latest `drush-launcher` release, currently `0.5.0`.  

Reason: I was getting [this error](https://github.com/drush-ops/drush/issues/3049) with Drush `9.0.0-rc1`, Drupal `8.4.2`, and the error went away after installing the latest `drush-launcher` per [greg-1-anderson's comment](https://github.com/drush-ops/drush/issues/3049#issuecomment-335459723).